src/rest/v1/openapi.yml

```diff
@@ -256,7 +291,7 @@ paths:
               schema:
                 type: object
                 properties:
-                  shifts:
+                  shiftAssignments:
```

Comment:
This is a breaking change for existing clients

Reply:
🔴

Reply:
Yup...caught this too.